src/mjolnir/landmark_builder.cc

```diff
@@ -183,10 +184,58 @@ void LandmarkDatabase::insert_landmark(const std::string& name,
   pimpl->vacuum_analyze = true;
 }
 
-std::vector<Landmark> LandmarkDatabase::get_landmarks_in_bounding_box(const double minLat,
-                                                                      const double minLong,
-                                                                      const double maxLat,
-                                                                      const double maxLong) {
+// get multiple landmarks by their ids
+std::vector<Landmark> LandmarkDatabase::get_landmarks_by_ids(const std::vector<int64_t>& pkeys) {
+  sqlite3_stmt* get_landmarks_by_ids_stmt = nullptr;
```

Comment:
we should put a TODO to test how this performs and see if we should prepare a default query (like 10-20 elements) and if more come in we extend the statement preparation

Reply:
done.